FreeRTOS/Demo/RISC-V_RV32_QEMU_VIRT_GCC/main.c

```diff
@@ -55,6 +55,9 @@
 #include <stdio.h>
 #include <string.h>
 
+/* Demo Config */
+#include "FreeRTOSConfig.h"
```

Comment:
Why this?

Reply:
I put mainVECTOR_MODE_DIRECT in FreeRTOSConfig.h.

Reply:
Once the CI supports to config the FreeRTOSConfig.h we can switch the vector/derict mode from CI by adding parameters or in a same way of selecting test cases.